scss/_variables.scss

```diff
@@ -932,6 +932,7 @@ $dropdown-header-padding:           $dropdown-padding-y !default; // Boosted mod
 // Pagination
 
 $pagination-margin-y:               $spacer !default;
+$pagination-margin-x:               .5rem !default; // Boosted mod
```

Comment:
Should be `.625rem`

Reply:
ah oui, j'ai gardé la même valeur sans vérifier sur le deisgn...